docs/source/checks/tabular/model_evaluation/plot_train_test_prediction_drift.py

```diff
@@ -26,7 +26,7 @@
 in which labels are not available for the test dataset, and so a drift in the predictions
 is our only indication that a changed has happened in the data that actually affects model
 predictions. If labels are available, it's also recommended to run the
-:doc:`Label Drift check </checks_gallery/tabular/train_test_validation/plot_train_test_label_drift>`.
+:doc:`Label Drift check </checks_gallery/tabular/train_test_validation/plot_label_drift>`.
```

Comment:
I think that in past deprecation we straight-out deleted the plot examples of the deprecated checks. Alternatively, this can lead to the new doc.

Reply:
@nirhutnik do you remember?

Reply:
Agree, no need to keep the old plot_... etc.

Reply:
Ok, removing all plot_train_test...

Reply:
Note that you have `examples_validation.py` in the tests, you need to add the deprecated checks to the `ignored_classes` in order to not check for their plot files.

But if someone will use the deprecated check he will be navigated to a broken link, so not sure what's correct...